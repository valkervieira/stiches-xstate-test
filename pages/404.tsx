import { Template404 } from '@templates/404';

export const Custom404 = (): JSX.Element => {
  return <Template404 />;
};

export default Custom404;
