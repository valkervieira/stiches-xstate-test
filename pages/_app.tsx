import type { AppProps } from 'next/app';
import { Session } from 'next-auth';
import { Provider as SessionProvider } from 'next-auth/client';
import { reset } from 'stitches-reset';
import { globalCss } from 'stiches.config';
import { Layout } from '@templates/Layout';

const globalStyles = globalCss(reset, { body: { background: '$white' } });

export const MyApp: (props: AppProps<{ session: Session }>) => JSX.Element = ({
  Component,
  pageProps: { session, ...pageProps },
}): JSX.Element => {
  globalStyles();

  return (
    <SessionProvider session={session}>
      <Layout>
        <Component {...pageProps} />
      </Layout>
    </SessionProvider>
  );
};

export default MyApp;
