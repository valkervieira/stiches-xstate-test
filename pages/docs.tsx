import { ConstructionTemplate } from '@templates/Construction';
import { Layout } from '@templates/Layout';
// import { DocsTemplate } from '@templates/Docs';

export const Docs = (): JSX.Element => {
  return (
    <Layout>
      <ConstructionTemplate />
    </Layout>
  );
};

export default Docs;
