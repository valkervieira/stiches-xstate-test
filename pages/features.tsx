import { H1 } from '@components/Text';

export const Features = () => {
  return <H1>Features page</H1>;
};

export default Features;
