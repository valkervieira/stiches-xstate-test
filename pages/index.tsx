import { HomeTemplate } from '@templates/Home';

export const Home = (): JSX.Element => {
  return <HomeTemplate />;
};

export default Home;
