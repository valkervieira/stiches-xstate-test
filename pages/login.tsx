import { Layout } from '@templates/Layout';
import { LoginTemplate } from '@templates/Login';

export const Login = (): JSX.Element => {
  return (
    <Layout>
      <LoginTemplate />
    </Layout>
  );
};

export default Login;
