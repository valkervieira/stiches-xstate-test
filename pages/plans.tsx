import { ConstructionTemplate } from '@templates/Construction';
import { Layout } from '@templates/Layout';
// import { PlansTemplate } from '@templates/Plans';

export const Plans = (): JSX.Element => {
  return (
    <Layout>
      <ConstructionTemplate />;
    </Layout>
  );
};

export default Plans;
