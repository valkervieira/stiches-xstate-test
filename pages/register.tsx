import { RegisterTemplate } from '@templates/Register';

export const Register = (): JSX.Element => {
  return <RegisterTemplate />;
};

export default Register;
