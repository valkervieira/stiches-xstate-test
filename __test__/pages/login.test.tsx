import React from 'react';
import { render } from '../testUtils';
import { Login } from '../../pages/login';

describe('Login page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Login />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
