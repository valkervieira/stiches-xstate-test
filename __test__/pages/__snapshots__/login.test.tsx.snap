// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page matches snapshot 1`] = `
<DocumentFragment>
  <main>
    <h1
      class="c-PJLV"
    >
      hey
    </h1>
  </main>
</DocumentFragment>
`;
